### ISSUE TITLE
Adopt views framework from SpatialEcology

### ISSUE THREAD

**Issue by kescobo** [BioJulia/Microbiome.jl#142]
For better filtering etc

**Comment by kescobo** [comment]
Duplicate of #60

**Comment by mkborregaard** [comment]
Yeah I hear you. Have not exactly been very good at maintaining this. sorry.

**Comment by kescobo** [comment]
It's not on you to maintain this package! I was just trying to come up with a list of things that need to be done before tagging 1.0, and this is a big one. But then I remembered there's already an issue for it :-)
